EmbeddedPkg/Library/PrePiLib/FwVol.c

```diff
@@ -814,12 +814,14 @@ FfsAnyFvFindFirstFile (
 EFI_STATUS
 EFIAPI
 FfsProcessFvFile (
-  IN  EFI_PEI_FILE_HANDLE  FvFileHandle
+  IN  EFI_PEI_FILE_HANDLE  FvFileHandle,
+  IN EFI_PEI_FV_HANDLE     ParentVolumeHandle  // MU_CHANGE

```

Comment:
nit: spacing. If doesn't trip uncrustify, doesn't matter

Reply:
Yeah, I defer to uncrustify and it's happy.